execution_engine_testing/tests/src/test/contract_api/transfer.rs

```diff
@@ -26,10 +28,19 @@ static TRANSFER_2_AMOUNT_WITH_ADV: Lazy<U512> = Lazy::new(|| *DEFAULT_PAYMENT +
 static TRANSFER_TOO_MUCH: Lazy<U512> = Lazy::new(|| U512::from(u64::max_value()));
 static ACCOUNT_1_INITIAL_BALANCE: Lazy<U512> = Lazy::new(|| *DEFAULT_PAYMENT);
 
-const ACCOUNT_1_ADDR: AccountHash = AccountHash::new([1u8; 32]);
-const ACCOUNT_2_ADDR: AccountHash = AccountHash::new([2u8; 32]);
+static ACCOUNT_1_SK: Lazy<SecretKey> =
```

Comment:
Please always avoid abbreviating public key and secret key to "pk" or "sk", even for tests.